cmd/insights/root.go

```diff
@@ -80,8 +80,8 @@ func init() {
 }
 
 func preRun(cmd *cobra.Command, args []string) {
-	// If this is the root or version command don't try to parse config
-	if cmd.Use == "insights" || cmd.Use == "version" {
+	// If  we do not need to contact Insights, don't try to parse config
+	if cmd.Use != "list" && cmd.Use != "sync" {
```

Comment:
🤔 I think this could potentially lead to confusion/bugs down the line. Also there's logic here for e.g. the logrus level.

Did you add this because there might not be a fairwinds-insights.yaml? Maybe we should just print a warning and move on if that's the case.

Reply:
I cleaned this up a bit. I have additional cleanup work planned for how we process config file / env vars / arguments, TBD RE: Cobra and Viper possibilities outside of this PR.